core/src/actions/navigation.ts

```diff
@@ -192,47 +192,50 @@ export class NavigationList extends OptionallyAuthenticatedAction {
       });
     }
 
-    if (navigationMode === "config") {
+    if (navigationMode === "config:unauthenticated") {
```

Comment:
This means that the user is able to navigate to `/apps`, `/profiles`, etc before they register.  Is this intentional?
<img width="1222" alt="Screen Shot 2021-06-09 at 8 41 41 AM" src="https://user-images.githubusercontent.com/303226/121386041-84313d80-c8fe-11eb-9a15-1a37e7786a9a.png">

Reply:
@evantahler The logic here is saying that if a config user is _not_ authenticated, all they see is a link to `/` (which would have the button to sign in). If they are authenticated, then they get the full nav.